ion_phys/ions/ca43.py

```diff
@@ -25,14 +25,12 @@
 
 
 class Ca43(Ion):
-    def __init__(self, B=None, *, level_filter=None, transition_filter=None):
+    def __init__(self, B=None, *, level_filter=None):
```

Comment:
unrelated note to self: I think it would be better to change the call signature here to be
```python
    def __init__(self, *, B=None, level_filter=None):
```
to force `B` to be a kwarg. I think that will make things clearer. e.g.
```python
Ca43(B=146e-4)
```